io.gitlab.librewolf-community.json

```diff
@@ -0,0 +1,84 @@
+{
+    "app-id": "io.gitlab.librewolf-community",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "40",
+    "sdk": "org.gnome.Sdk",
+    "command": "librewolf",
+    "rename-appdata-file": "io.gitlab.librewolf-community.appdata.xml",
+    "rename-icon": "io.gitlab.librewolf-community",
```

Comment:
You probably want to do the renames upstream if possible

Reply:
Fair point. IIRC that's still there as an artifact from some ancient shenanigans where things weren't found during validations although there (somewhat related to the odd issue where the icon file isn't found if `usr` is also used). Could probably be dropped as well, I'll give it a try.

Thanks for all the feedback, btw. – the whole manifest was probably quite a bit messier than it could've been owed to it previously being used in a rather messy manual build process. :)

Reply:
Could be removed without any ill effects.